src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassFanOutComplexityCheck.java

```diff
@@ -48,10 +48,12 @@
  * <ul>
  * <li>
  * Property {@code max} - Specify the maximum threshold allowed.
+ * Datatype - {@code Integer}
  * Default value is {@code 20}.
  * </li>
  * <li>
  * Property {@code excludedClasses} - Specify user-configured class names to ignore.
+ * Datatype - {@code String}
```

Comment:
it is not string, it is string set

Please try to keep it the same as the docs for properties, e.g. https://checkstyle.org/property_types.html#Overview

Reply:
https://checkstyle.org/config_metrics.html#ClassFanOutComplexity_Properties

Text on xdoc is generated from https://github.com/checkstyle/checkstyle/blob/master/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java#L862 . Some properties require an override.